packages/wagmi/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lens-protocol/wagmi",
-  "version": "2.1.0-next.9",
+  "version": "3.0.0-next.10",
```

Comment:
2.1.0-next.10

Reply:
Great catch... didn't add it to staged files 🤦